Sniff.php

```diff
@@ -953,4 +953,50 @@ public function getMethodParameters(PHP_CodeSniffer_File $phpcsFile, $stackPtr)
 
     }//end getMethodParameters()
 
+
+    /**
+     * Get the hash algorithm name from the parameter in a hash function call.
+     *
+     * @param PHP_CodeSniffer_File $phpcsFile Instance of phpcsFile.
+     * @param int                  $stackPtr  The position of the T_STRING function token.
+     *
+     * @return string|false The algorithm name without quotes if this was a relevant hash
+     *                      function call or false if it was not.
+     */
+    public function maybeGetHashAlgorithmParameter(PHP_CodeSniffer_File $phpcsFile, $stackPtr)
```

Comment:
Done ;-)

Also had a look at further reducing the duplicate code, but will leave that to another PR which will tackling that for all sniffs checking for function calls.